tests/periph_i2c/tests/base_if.py

```diff
@@ -0,0 +1,173 @@
+import serial
+import logging
+import errno
+import os
+import time
+import serial.tools.list_ports
+
+
+class IfParams:
+    cmd = ''
+    result = ''
+    msg = ''
+    data = ''
+
+
+class BaseIf:
+    __READ_REG_CMD = "rr"
+    __WRITE_REG_CMD = "wr"
+    __EXECUTE_CMD = "ex"
+    __RESET_CMD = "mcu_rst"
+    __SUCCESS = '0'
+    __RESULT_SUCCESS = 'Success'
+    __RESULT_ERROR = 'Error'
+    __RESULT_TIMEOUT = 'Timeout'
+
+    def __init__(self, port='/dev/ttyACM0', baud=115200):
+        self.connect(port, baud)
+
+    def get_port(self):
+        if (self.__dev.isOpen()):
```

Comment:
here (and below) `.isOpen()` is deprecated according to documentation, use `.is_open` instead